e2e-scripts/run-e2e.sh

```diff
@@ -48,6 +57,20 @@ if [ ! -f ${DOCKERCONFIGJSON} ]; then
   exit 1
 fi
 
+if [ -z ${SIPPY_LOAD_TIMEOUT} ]; then
+  # Allow timeout override in case it takes longer.
+  echo "SIPPY_LOAD_TIMEOUT is not set; assuming 300s"
+  export SIPPY_LOAD_TIMEOUT='300s'
```

Comment:
Hm... I see you define `SIPPY_LOAD_TIMEOUT` here but it's blank in the script that's. called later.

Reply:
there's this syntax in bash where you can define a variable to be something if it's not already defined.  Let me find a ref.

Reply:
Is that not similar to how you have DOCKERCONFIGJSON setup?

Reply:
yea, that is similar. hm...

Does this work when running it locally with both defining `SIPPY_LOAD_TIMEOUT` and not? it does seem like it should be defined.

Reply:
ah, I think I know why -- that `run-e2e.sh` is not run in CI.

Reply:
To get past the error in the CI, you'll need to define that new variable [in release repo](https://github.com/openshift/release/blob/5fb1845e3e2d5f75488d97e8cc94e760818a0971/ci-operator/step-registry/sippy-e2e/sippy-e2e-setup/sippy-e2e-sippy-e2e-setup-ref.yaml).

Reply:
Thanks, I just added defaults in the scripts.  Thanks for pointing out that run-e2e.sh wasn't getting called via CI.